cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -248,7 +248,7 @@ public void execute() {
                 //// check if the given file exists.
                 if (Files.notExists(sourcePath)) {
                     CommandUtil.printError(this.errStream,
-                            "'" + sourcePath + "' ballerina file does not exist.",
+                            "file not found: '" + sourcePath,
```

Comment:
This will print something like the following, without the ending `'`
```cmd
file not found: 'foo.bal
```

```suggestion
                            "file not found: '" + sourcePath + "'",
```

Reply:
Fixed.